lib/whitehall_importer/create_migration.rb

```diff
@@ -45,6 +45,7 @@ def create_document_imports(page, record)
         record.document_imports.create!(
           whitehall_document_id: document["document_id"],
           state: "pending",
+          content_id: document["content_id"],
```

Comment:
As per my comment in the `index_document_factory`, this will need to be obtained from the `document_information`:
```suggestion
          content_id: document["document_information"]["content_id"],
```